parallaxinc/Parallax-IDE
# replace browserify-fs with chrome-sync-fs

#### What's this PR do?

Replaces `browserify-fs` with `chrome-sync-fs` during the bundle process.  This allows us to continue to use the workspace implementation as it currently stands, including temp file support using IndexedDB.  The workspace abstraction still needs work before it will be ready to ship.
#### Where should the reviewer start?

`npm install` and `npm run build`
#### How should this be manually tested?

Verify that the application still works (you might need to clear the user config).  Create some files, add some text, jump between files.  Then check to see if the files are saved in Google Drive.
#### Any background context you want to provide?

There is a larger task of re-architecting the workspace abstraction, but unforeseen complexity has caused it to not make this release.  This is a hold-over to get `syncFileSystem` support in the editor on this sprint.
#### What are the relevant tickets?

Closes #84
